setup.py

```diff
@@ -1,5 +1,16 @@
 # coding: utf-8
-from setuptools import setup, find_packages
+from setuptools import setup, find_packages, Extension
+
+FLATCC_PREFIX='/opt/circonus'
```

Comment:
This will put a dependency on our flatcc package.  Might be OK? @esproul opinions?